build.sbt

```diff
@@ -157,3 +135,26 @@ lazy val pine = (projectMatrix in file("."))
   .jsPlatform(     Seq(V.scala2_11), JsSettings     ++ Scala11Settings)
   .nativePlatform( Seq(V.scala2_11), NativeSettings ++ Pre13Settings)
 
+// root settings. src/ is handled by sbt-projectmatrix
+publish / skip := false
```

Comment:
Added these since you are not using the root project.

Reply:
Great. I was wondering whether this was possible.